internal/image/fetcher.go

```diff
@@ -7,15 +7,17 @@ import (
 	"io"
 	"strings"
 
-	"github.com/buildpacks/imgutil"
-	"github.com/buildpacks/imgutil/local"
-	"github.com/buildpacks/imgutil/remote"
-	"github.com/buildpacks/lifecycle/auth"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/jsonmessage"
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/buildpacks/imgutil"
```

Comment:
nit: the convention in this repo is to group imports from `buildpacks/*` with the external imports unless they are from `/buildpacks/pack/*`

Reply:
Thanks! Force pushed the formatting